src/Encryption/CodecServer/Program.cs

```diff
@@ -0,0 +1,62 @@
+namespace TemporalioSamples.Encryption.CodecServer;
+
+using Google.Protobuf;
+using Temporalio.Api.Common.V1;
+using Temporalio.Converters;
+using TemporalioSamples.Encryption.Codec;
+
+public sealed class Program
+{
+    public static void Main(string[] args)
+    {
+        var builder = WebApplication.CreateBuilder(args);
+
+        // Setup console logging, codec, and cors
+        builder.Logging.AddSimpleConsole().SetMinimumLevel(LogLevel.Information);
+        builder.Services.AddSingleton<IPayloadCodec>(ctx => new EncryptionCodec());
+        builder.Services.AddCors();
+
+        var app = builder.Build();
+
+        // We need CORS so that the browser can access this endpoint from a
+        // different origin
+        app.UseCors(
+            builder => builder.
+                WithHeaders("content-type", "x-namespace").
+                WithMethods("POST").
+                WithOrigins("http://localhost:8080", "https://cloud.temporal.io/"));
```

Comment:
Might want to put `8233` here for the CLI's web UI and document that users may need to customize these URLs.

Reply:
:+1: Will do

Reply:
Fixed